modules/configuration/pages/error_handling.adoc

```diff
@@ -95,6 +100,24 @@ pipeline:
           root.meta.error = error()
 ----
 
+ifndef::env-cloud[]
+=== Logging fatal errors
+
+During development and testing, you can use the xref:components:processors/crash.adoc[`crash` processor] to halt pipeline execution when an unhandled error occurs and log a fatal message. Use Bloblang functions to customize the message. For example:
+
+[source,yaml]
+----
+pipeline:
+  processors:
+    - try:
+      - resource: foo # Processor that might fail
+      - resource: bar # Processor that might fail
+      - resource: baz # Processor that might fail
+    # Writes a custom fatal log message
+    - crash: "Processor ${!error_source_label()} failed due to: ${!error()}"
```

Comment:
This should be behind a `catch` processor otherwise this always fails, see the example above in this file:

```
pipeline:
  processors:
    - resource: processor_1 # Processor that might fail
    - catch:
      - crash: "processing failed at ${!error_source_label()} due to: ${!error()}"
```